CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+
+### **What do I need to know to help?**
+
+If you are looking to help to with a code contribution, our project uses Go Lang. If you don't feel ready to make a code contribution yet, no problem! You can also check out the documentation <https://github.com/score-spec/docs>.
+
+If you are interested in making a code contribution and would like to learn more about the technologies that we use, check out the list below.
+
+- [https://humanitec.com](https://humanitec.com/)
+- [https://docs.docker.com/compose/](https://docs.docker.com/compose/)
+- [https://helm.sh/docs/](https://helm.sh/docs/)
+
+### **How do I make a contribution?**
+
+Never made an open-source contribution before? Wondering how contributions work in our project? Here's a quick rundown!
+
+1. Find an issue that you are interested in addressing or a feature that you would like to add.
+2. Fork the repository associated with the issue to your local GitHub organization. This means that you will have a copy of the repository under **your-GitHub-username/repository-name**.
+3. Clone the repository to your local machine using **git clone**.
+4. Create a new branch for your fix using **git checkout -b your-branch-name**.
+5. Make the appropriate changes for the issue you are trying to address or the feature that you want to add.
+6. Use **git add insert-paths-of-changed-files-here** to add the file contents of the changed files to the "snapshot" git uses to manage the state of the project, also known as the index.
+7. Use **git commit -s -m "Insert a brief message of the changes made here"** to store the contents of the index with a descriptive message.
+8. Push the changes to the remote repository using **git push origin your-branch-name**.
+9. Submit a pull request to the upstream repository.
+10. Title the pull request with a brief description of the changes made and the issue or bug number associated with your change. For example, you can title an issue like so, "Added more log outputting to resolve #4352".
+11. In the description of the pull request, explain the changes that you made, any issues you think exist with the pull request you made, and any questions you have for the maintainer. It's OK if your pull request is not perfect (no pull request is), the reviewer will be able to help you resolve any problems and improve it!
+12. Wait for the pull request to be reviewed by a maintainer.
+13. Introduce changes to the pull request if the reviewing maintainer recommends them.
+14. Celebrate your success after your pull request is merged!
+
+## Feature requests
+
+## Code reviews
+
+All submissions, including submissions by project members, require review.
+
+Score uses GitHub pull requests for this purpose. 
+
+The general workflow for code contributions:
+
+1. Submit/find an issue in this repository.
```

Comment:
It would also make sense to mark the issue as in progress to avoid duplicate work. This can be done by dropping in a comment or linking a PR to it as done here for example: https://github.com/score-spec/score-humanitec/pull/18